frontend/src/metabase/plugins/index.ts

```diff
@@ -197,7 +197,6 @@ export const PLUGIN_LDAP_FORM_FIELDS = {
     settings: {
       [setting: string]: {
         display_name?: string | undefined;
-        warningMessage?: string | undefined;
```

Comment:
this was only ever used by one setting, and I got rid of this bit of specialness